pkg/display/display.go

```diff
@@ -139,11 +139,9 @@ func getColumnSettings(headers []string, rows [][]string, opts *ShowWrappedTable
 
 	// now that all columns are set to the widths that they need,
 	// set the last one to occupy as much as is available - no more - no less
-	const MaxWidth = 200
 	sumOfRest := sumOfAllCols - colConfigs[len(colConfigs)-1].WidthMax
-	widthConstraint := utils.NewRangeConstraint(sumOfAllCols, MaxWidth)
 	// get the max cols width
-	maxCols := GetMaxCols(widthConstraint)
+	maxCols := GetMaxCols()
```

Comment:
this no longer takes `widthConstraint` into account?

Reply:
No. This was never really necessary here. `GetMaxCols` returns the maximum available columns.

We use the `constrain` in `check` `table format`. There we have moved the check one-level up.